### DISCUSSION TITLE
unable to use tasks' output parameter

### DISCUSSION THREAD

**Discussion by prateekatam** [argoproj/argo-workflows#13686] [Q&A]
I'm trying to run a simple workflow using https://argo-workflows.readthedocs.io/en/latest/walk-through/output-parameters/#output-parameters

It doesn't work as expected. In the `print-message` step logs, I see `{{steps.generate-parameter.outputs.parameters.hello-param}}` instead of `hello world`

in the Argo workflows UI, the inputs/outputs has:

for generate-parameter : Inputs > Parameters-  no parameters | Outputs > Parameters- no parameters
for consume-parameter: Inputs > Parameters-  `message: message
{{steps.generate-parameter.outputs.parameters.hello-param}}`
  | Outputs > Parameters- no parameters

looks like generate parameter is not producing an output parameter. What could be going wrong? TIA

Can't upload screenshot unfortunately :(

**Comment by agilgur5** [comment]
This might just be a bug: https://github.com/argoproj/argo-workflows/issues/13619
